### DISCUSSION TITLE
How to change the number of issues per page in the Github issue tracker?

### DISCUSSION THREAD

**Discussion by amelvill-umich** [community/community#23689] [New to GitHub]
<p>I was wondering, is it possible to change the number of issues displayed per page on a github repo’s issue page? Maybe something in user preferences?</p>
<p>I noticed that there seems to be some way to do it via the github API, but I was wondering if there was a way to do this on <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> just by adding something to the end of the issues link, or something like that.</p>
<p>If this doesn’t exist, here’s something like a feature request, if it helps:</p>
<ul>
<li>If possible, I would like to display 100, 1000 or all the issues on a single page. Anything to increase the number of issues per page would be welcome.</li>
<li>I would be OK with a very barebones issue page, I would prefer it if it wasn’t “infinitely scrolling” like Youtube’s search results that loaded more issues only when you scrolled down. I’m alright with even reading a plain text list of issue links.</li>
<li>The reason I don’t like infinite scrolling is because sometimes I want to use Ctrl+F in my browser to find keywords or parts of keywords, and also the noticeable lag when I am scrolling through looking for something, but can’t think of an exact search term.</li>
<li>If there’s no way to do that without killing the Github server because the API wasn’t designed for it, I understand.</li>
</ul>
<p>I guess I could look into making a script to do this. if it’s not possible.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/amelvill-umich">@amelvill-umich</a>!</p>
<p>It is not currently possible to change how Issues are displayed in this way on <a href="http://GitHub.com">GitHub.com</a>. The API does allow you to change how many issues are pulled, so you could write a script to interact with the API in that way to build a custom interface allowing you to get the information and customization that you want.</p>
<p>If you’d rather log this as a feature request, I recommend using the <a href="https://support.github.com/contact/feedback">official GitHub feedback form</a> which goes to our product and engineering teams. You can always add a link to this forum topic to reduce how much you have to copy and paste into the feedback form if you wish.</p>
<p>Thanks!</p>

**Comment by amelvill-umich** [comment]
<p>Thanks for the reply, I have submitted feedback using the form.</p>

**Comment by wiejakp** [comment]
is there any update on this issue? would be nice to have ability to list all PR's closed from this month in one page so I can export it for my manager.

**Comment by monperrus** [comment]
This could be a feature of [refined-github](https://github.com/refined-github/refined-github/)

But the maintainers don't want it, see https://github.com/refined-github/refined-github/issues/7326

The only option would be to write a specific userscript or browser extension.
